File: 2015/gara3/taxi/att/taxi.c
#include <stdio.h>
#include <assert.h>

#define MAXN 10000

int viaggia(int N, int C[]) {
    // Mettete qui il codice della soluzione
    return 42;
}


int C[MAXN];

int main() {
    FILE *fr, *fw;
    int N, i;

    fr = fopen("input.txt", "r");
    fw = fopen("output.txt", "w");
    assert(1 == fscanf(fr, "%d", &N));
    for(i=0; i<N; i++)
        assert(1 == fscanf(fr, "%d", &C[i]));

    fprintf(fw, "%d\n", viaggia(N, C));
    fclose(fr);
    fclose(fw);
    return 0;
}

File: ed1/gara3/taxi/att/taxi.pas
function viaggia(N: longint; var C: array of longint): longint;
begin
    (* Mettete qui il codice della soluzione *)
    viaggia := 42;
end;


const
    MAXN = 10000;
var
    N, i  : longint;
    C     : array[0..MAXN-1] of longint;
    fr, fw: text;

begin
    assign(fr, 'input.txt');
    assign(fw, 'output.txt');
    reset(fr);
    rewrite(fw);
    readln(fr, N);
    for i:=0 to N-1 do
        read(fr, C[i]);

    writeln(fw, viaggia(N, C));
    close(fr);
    close(fw);
end.
